portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,102 @@
+html, body {
+  height: 100%;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
-#greeting-container {
+.page {
+  height: 100%;
+  margin: 0;
+  overflow: hidden;
+}
+
+h1 {
+  font-size: 500%;
+  text-align: center;
+}
+
+h2 {
+    font-size: 250%;
+}
+
+#home {
+  background-image: url('/images/toba-1-crop.jpg');
+  background-repeat: no-repeat;
+  background-size: 100%; 
+  padding-top: 3%;
+}
+
+#about {
+  background-image: url('/images/toba-memeoji2.jpg');
+  background-position: 80% 40%;
+  background-repeat: no-repeat;
+  background-size: 35%;
+}
+
+#about h2, p {
+  font-size: 2vw;
+  margin-left: 10%;
+  max-width: 40%;
+}
+
+#quote-container {
+  font-size: 2vw;
+  margin-left: 10%;
   margin-top: 20px;
+  max-width: 40%;
 }
+
+.stack {
+  width: 100%;
+}
+
+#life {
+  background-color: #5e5d48;
+}
+
+.extended-page {
+  margin-top:10%;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 4px;
+}
+
+/* Create four equal columns that sits next to each other */
+.column {
+  flex: 25%;
+  max-width: 24.24%;
```

Comment:
Not necessary to change for this, but did you consider using calc(25% - 8px) where you can guarantee that width of the column will keep room for 4px * 2 of side spacing? Might not be needed if your bootstrap changes the default box-sizing, but consider it if that's what led you to define the 24.24% max width.

Reply:
wow....you're better than w3schools lol. I did 24.24% because when I did 25%, one row would overflow to the bottom. But when I did 24% the margin on the right would be a lot bigger. So i was trying to cheat with the .24. But why does the minus 8 work and what exactly is it doing?

Reply:
Haha woo! Well, I'll tell you what I thought it was - I thought that you still had the default `box-sizing: content-box` which would mean that your 25% max-width turns into 25% + 4px + 4px with your side paddings. Buuuut, it looks like your bootstrap should be defaulting everything to border-box, so it would seem that what you had should work (box-sizing: border-box; flex: 25%; max-width: 25%; should work in theory). Did you by chance add in the bootstrap after testing this, or can you confirm that it's still an issue if you return max-width to 25%?

Reply:
yes , 25% does not work

Reply:
That's fine - sometime later I may grab your code and see what's going on so we can talk about it more.